socorro/signature/rules.py

```diff
@@ -58,33 +58,12 @@ def action(self, crash_data, result):
         return True
 
 
-class SignatureTool:
```

Comment:
This suggested that subclasses followed a standard API. There are two subclasses and they both had very different `_do_generate` methods and thus, there was no standard API. Given that, I removed this untruthful thing since it's not helping any.